plugins/katello/3.9/release_notes/release_notes.md

```diff
@@ -19,6 +19,10 @@ For Red Hat Customers we've introduced a new feature called System Purpose to im
 ### Content View Import/Export
 Katello 3.9 ships with a new content view version import/export feature. This feature is meant to replace the inter-server synchronization feature. It allows users to export and import content view versions along with associated yum content. Please see the [instructions](https://www.theforeman.org/plugins/katello/nightly/user_guide/content_view_import_export/index.html) for more details.
 
+### Katello Client Repo Changes
+The Katello client repo has been taken into the Foreman client repo. The docs were originally incorrect, but have since been updated.
```

Comment:
Maybe jazz this up a bit, something including:

 * katello client repos have been removed in favor of a Foreman client repo (they can be found at http://yum.theforeman.org/client/1.20/)
 * please see install and upgrade instructions for clients to switch